<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>E4t3</class>
 <widget class="QWidget" name="E4t3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="vecWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="matWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="vecWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>189</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>插值方法</string>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>分段线性插值</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>171</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>分段3次Hermite插值</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>三次样条插值</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>vecWidget</class>
   <extends>QWidget</extends>
   <header>vecwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>matWidget</class>
   <extends>QWidget</extends>
   <header>matwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
